Dockerfiles/Dockerfile-ubuntu-18.04

```diff
@@ -4,38 +4,28 @@ FROM ubuntu:18.04
 RUN \
   apt-get update && \
   apt-get -y upgrade && \
-  apt-get install -y apt-utils autoconf bison build-essential byobu curl default-jre emacs \
+  apt-get install -y apt-utils autoconf bison build-essential byobu curl default-jre dialog emacs \
     fftw3-dev flex gdb g++-8 gcc-8 gfortran git git-lfs htop libblas-dev libcurl4-gnutls-dev \
     libpugixml-dev libcfitsio-dev libgtest-dev libhdf5-dev liblapack-dev libncurses-dev \
     libreadline-dev libssl-dev libstarlink-ast-dev libtbb-dev libtool \
     libxml2-dev libzstd-dev libgsl-dev man pkg-config python-pip python3-pip \
-    software-properties-common unzip vim wcslib-dev wget cmake uuid-dev && \
+    software-properties-common unzip vim wcslib-dev wget uuid-dev && \
   update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 80 --slave /usr/bin/g++ g++ /usr/bin/g++-8 --slave /usr/bin/gcov gcov /usr/bin/gcov-8
 
-# Get carta dependencies
-# casacore data from Kernsuite PPA
+# carta_backend unit tests require cmake >3.12 but ubuntu 18.04 provides 3.10.2
 RUN \
-  apt-add-repository -y -s ppa:kernsuite/kern-5 && \
-  apt-get update && \
-  apt-get -y install casacore-data
```

Comment:
Our PPA does not package casacore-data. We can get this either from KERN or from the main Ubuntu repository. I assume that there's a reason that we've been using the KERN version, so we should probably keep using it for consistency.

Reply:
It may be unnecessary to install casacore-data explicitly here; it should be installed automatically later as a recommended dependency of our casacore package. But we should check this; I've seen that in some environments apt is configured not to install recommended dependencies by default.

Reply:
I've re-added the KERN PPAs to these two docker files. We can also use the standard Ubuntu version of casacore-data, but that has to pull in a bunch of other casacore dependencies because of how it's structured.